### PR TITLE
add lastupdated to case entity to keep in sync with case processor

### PR DESCRIPTION
**Pull request by hbrace**
# Motivation and Context

Adding this lastUpdated time stamp will provide the capability to do 'delta' based extracts from RM for data that is sent to DAP for use by processing and the RCA/FPA algorithms as opposed to exported the full set each time.

# What has changed

This is to match the changes to the Case entity that are in the case processor on a branch of the same name.

# How to test?

Run maven clean install after running maven clean install on the caseprocessor branch.
# Links

https://trello.com/c/YHny5NEN/223-add-lastupdated-field-to-case-db-cases-table-3
# Screenshots (if appropriate):

### CODE REVIEW

**Comment by hbrace** [comment]
[add "lastUpdated" field to Case DB cases table (3)](https://trello.com/c/YHny5NEN/223-add-lastupdated-field-to-case-db-cases-table-3)

File: src/main/java/uk/gov/ons/census/caseapisvc/model/entity/Case.java [hunk 1]
```diff
@@ -115,4 +116,6 @@
 
   @Column(nullable = false, columnDefinition = "BOOLEAN DEFAULT false")
   private boolean undeliveredAsAddressed;
+
+  @Column @UpdateTimestamp private OffsetDateTime lastUpdated;
```
Comment (ryangrundy7) [inline src/main/java/uk/gov/ons/census/caseapisvc/model/entity/Case.java]: `(columnDefinition = "timestamp with time zone")` need this in here as well

Comment (hbrace) [reply src/main/java/uk/gov/ons/census/caseapisvc/model/entity/Case.java]: 👍 added

**Review by ryangrundy7 — APPROVED** [review]

**Review by NickGrantX — APPROVED** [review]
LGTM 👍

**Pull request merged**